package.json

```diff
@@ -30,13 +30,15 @@
     "build-ts-api": "tsc --emitDeclarationOnly && cpy dist/web-ifc-api.d.ts dist && cpy dist/web-ifc-api.d.ts dist --rename=web-ifc-api-node.d.ts",
     "build-web-ifc-api-mjs": "esbuild dist/web-ifc-api.ts --bundle --format=esm --external:path --external:fs --outfile=./dist/web-ifc-api.js",
     "build-web-ifc-api-node": "esbuild dist/web-ifc-api.ts --bundle --platform=node --outfile=./dist/web-ifc-api-node.js",
-    "copy-to-dist": "cpy README.md dist && cpy package.json dist && cpy src/ifc2x4.ts dist",
+    "copy-to-dist": "cpy README.md dist && cpy package.json dist && cpy src/ifc2x4.js dist",
```

Comment:
Is this still needed?

Reply:
Fixed!

Reply:
I'm still seeing it. Am I missing something? @agviegas

Reply:
Woops, my bad. I didn't push.

Reply:
Ready!